.github/workflows/test.yml

```
@@ -0,0 +1,68 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
```

This would be useful:
```suggestion
    strategy:
      fail-fast: false
```

At the moment, if a single job fails, the rest are all auto-cancelled.

This can be especially annoying for temporary time-outs. It was also a pain for my other PR: first 3.3 failed because it wasn't supported, which cancelled the 2.7 job, and masking that I also needed to pin dependencies for 2.7. 

Turning it off allows the other jobs to finish running, which can help you determine if the failure is a compatibility bug (e.g. fails on 2.7, passes on all 3.x) or a more common coding bug (fails on all).

The default auto-cancelling behaviour can be useful, but I find best for when there's a particularly large and slow-running matrix for a busy repo.

Otherwise for quick builds, like most of these (e.g. only 2 minutes here), I suggest letting them all run.